### PR TITLE
Update Newtonsoft.Json to 12.0.1

### PR DESCRIPTION
**Pull request by indualagarsamy**
Fixes https://github.com/Particular/NServiceBus.Persistence.Sql/issues/322

### CODE REVIEW

File: src/ScriptBuilder.Tests/ScriptBuilder.Tests.csproj [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit" Version="[3.11.0, 4.0.0)" />
```
Comment (bording) [inline src/ScriptBuilder.Tests/ScriptBuilder.Tests.csproj]: No need for version ranges on projects that aren't released as a package.

File: src/ScriptBuilderTask.Tests/ScriptBuilderTask.Tests.csproj [hunk 2]
```diff
@@ -14,9 +14,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit" Version="[3.11.0, 4.0.0)" />
```
Comment (bording) [inline src/ScriptBuilderTask.Tests/ScriptBuilderTask.Tests.csproj]: No need for version ranges on projects that aren't released as a package.

File: src/SqlPersistence.Tests/SqlPersistence.Tests.csproj [hunk 3]
```diff
@@ -15,9 +15,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit" Version="[3.11.0, 4.0.0)" />
```
Comment (bording) [inline src/SqlPersistence.Tests/SqlPersistence.Tests.csproj]: No need for version ranges on projects that aren't released as a package.

**Comment by indualagarsamy** [comment]
Closing this PR, as the release branch was created incorrectly. Will open a new one.

**Pull request closed without merging**